### PR TITLE
feat :added alert component

### PR DESCRIPTION
**Pull request by Tanjum786**
_**Created Alert component:**_

Added different types of alerts like :

- Primary alert 
- Secondary alert
- Warning alert 
- Success alert 
- Danger alert

Screenshot from 2022-02-04 23-26-37

### CODE REVIEW

File: component/alerts/alert.css [hunk 1]
```diff
@@ -0,0 +1,57 @@
+.alert_container {
+    border-radius: 0.5rem;
+    background-color: var(--primary-alert-color);
+    font-size: 1.5rem;
+    height: 4rem;
+    padding: 1rem;
+    width: 50rem;
+}
+
+#Alerts .component_box {
```
Comment (aishwarya-bm) [inline component/alerts/alert.css]: It's better to use class selectors than id selectors.

File: component/alerts/alert.css [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+    text-decoration: none;
+}
+
+.alert_primary:hover {
+    color: var(--primary-color);
+}
+
+.alert_secondary {
+    background-color: #e0e7ff;
+}
```
Comment (aishwarya-bm) [inline component/alerts/alert.css:30]: Using variable names is recommended as we can reuse them in other places too, otherwise the code looks good!

Comment (Tanjum786) [reply component/alerts/alert.css:30]: Thank you. I will do this.

File: component/alerts/alert.css (hunk 1 above)
Comment (Tanjum786) [reply component/alerts/alert.css]: ok I will change.

**Review by anamkaa — COMMENTED** [review]
Everything looks good, can proceed to merge.

**Pull request merged**